### DISCUSSION TITLE
兼容性

### DISCUSSION THREAD

**Discussion by moonlitusun** [ant-design/ant-design-mobile#4550] [Q&A] [answered]
请问是否有测试过能兼容到安卓和ios的那个系统版本。比如webview Android

**Comment by awmleer** [comment, marked as answer]
关于 v5 版本的兼容性说明在这里哈：https://mobile.ant.design/zh/guide/faq

**Reply by moonlitusun** [reply]
你好，这个链接我看过，但是没有发现有兼容性的描述。
我知道上面的快速入门里面是有一个兼容性的说明。但是只说了可以配置iOS Safari >= 10 和 Chrome >= 49，问下有安卓的吗。
因为我想确认下安卓6.0是否可以使用

**Reply by awmleer** [reply]
噢之前是这个链接，现在在快速上手里了

**Reply by awmleer** [reply]
安卓的兼容性其实就是 Chrome，因为不像 iOS，Android 版本和其中的浏览器的 Chrome 版本不是绑定在一起的，所以 Android 6 这个说法其实并不是很有意义，我们就统一用 Chrome 版本来描述了

**Comment by WuHuSa** [comment]
你好，想接着问一下，官网提到的 chrome 从 49 开始兼容，是否代表安卓最低兼容到 chrome49 版本？是否用了 49 以下不兼容的 api 如 proxy 之类？

**Reply by awmleer** [reply]
用到了 CSS 变量，而 CSS 变量是从 Chrome 49 才开始支持的

**Reply by WuHuSa** [reply]
想了解下，兼容到 chrome49 这个结论的决策依据是什么？
是为了兼容技术侧「css变量」定的；还是说有充足的数据统计表明当前在 chrome49 以下的用户量很低，可以忽略？

**Reply by awmleer** [reply]
是因为我们用了 CSS 变量，而小于 49 版本的 Chrome 不支持 CSS 变量，所以无论怎样做 ployfill，都没办法超过这个兼容性极限了

**Reply by WuHuSa** [reply]
感谢回复！这样的话我就有了新的疑惑，望指点
「谁在使用」中列举的支付宝、淘宝，这些业务方一般都有对 android 的兼容要求范围（比如从 android5 兼容起，此时 chrome 是 37），此时使用 antd-mobile5，就会存在 chrome38-48 这些版本的不兼容，这样业务就无法达到兼容期望。
请教下这种情况业务方（如支付宝）需要怎么处理？
（另，刚发现我在 hox 那边提的 issuse 也是老兄你跟进解决的，缘分啊~  谨代表我们团队再次感谢贵团队给社区的贡献 🌺🌺🌺 要是能留个联系方式后面深入交流下就更好了）

**Reply by awmleer** [reply]
淘宝、支付宝这些部门的业务不仅仅是只有他们本身的 app，还包括了大量其他相关的业务，这其中很多业务对兼容性是没有那么严苛的要求的。而且即便是淘宝和支付宝 app 本身，也不是所有模块的兼容性要求都是一样的。
此外，Android 5 对应的支付宝 app（淘宝同理）中的浏览器内核并非系统原生的浏览器，而是 UC 内核，因此你所根据 Android 版本号推断的 Chrome 版本号在这里并不适用。关于支付宝 app 中浏览器内核兼容性相关的更详细的分析，可以参考[这篇分享](https://mp.weixin.qq.com/s/V28zHkCJFtzZskO-BP3UUA)。

至于联系方式，可以随时通过我 GitHub profile 上留的邮箱地址联系到我哈。
